test/integration/aaa_download_only_test.go

```diff
@@ -158,6 +158,14 @@ func TestDownloadOnly(t *testing.T) {
 				}
 			})
 
+			t.Run("minikube logs", func(t *testing.T) {
+				ctx, cancel := context.WithTimeout(context.Background(), Seconds(5))
+				args := append([]string{"logs", "-p", profile})
+				if _, err := Run(t, exec.CommandContext(ctx, Target(), args...)); err != nil {
+					t.Errorf("minikube logs failed with error: %v", err)
+				}
```

Comment:
Check explicitly for if context timed out and add a better error.
Or you could measure how long it took and say
"Minikube logs expected to finish by ... But took ...

Reply:
Added